diff --git a/node_modules/tsup/.DS_Store b/.DS_Store
new file mode 100644
index 0000000000000000000000000000000000000000..1f4bec7e0f55f25c287c38006447b82ea2583a80
Binary files /dev/null and b/.DS_Store differ
diff --git a/dist/rollup.js b/dist/rollup.js
index e128b61b9558318b9f86dc11d72c31f09a8eb7db..5068f0918d96c62e06ecb7b2113c717da569baae 100644
--- a/dist/rollup.js
+++ b/dist/rollup.js
@@ -6834,7 +6834,6 @@ var getRollupConfig = async (options) => {
           tsconfig: options.tsconfig,
           compilerOptions: {
             ...compilerOptions,
-            baseUrl: compilerOptions.baseUrl || ".",
             // Ensure ".d.ts" modules are generated
             declaration: true,
             // Skip ".js" generation
